public_html/wp-content/mu-plugins/blocks/assets/src/organizers/edit.js

```diff
@@ -0,0 +1,124 @@
+/**
+ * External dependencies
+ */
+import { isUndefined, pickBy, split } from 'lodash';
+
+/**
+ * WordPress dependencies
+ */
+const apiFetch                = wp.apiFetch;
+const { withSelect }          = wp.data;
+const { Component, Fragment } = wp.element;
+const { addQueryArgs }        = wp.url;
+
+/**
+ * Internal dependencies
+ */
+import OrganizersBlockControls     from './block-controls';
+import OrganizersInspectorControls from './inspector-controls';
+import OrganizersToolbar           from './toolbar';
+
+export const ORGANIZERS_ICON = 'groups';
```

Comment:
@coreymckrill , @vedanshujain : what do you think about this?

It feel odd to me that in the other blocks the icon is duplicated in `index` and `edit`, instead of being DRY, so this is my attempt to fix that.

I first tried to define it in `index` and pass it to `edit`, since that feels more logical, but didn't see a good way. This works, but feels a bit odd that it's being passed from `edit` to `index`.

Here's the usage in `index`:

https://github.com/WordPress/wordcamp.org/pull/33/commits/6560472e4740813d7bbf66613a625d417220432f#diff-bb376325754a536e359c629d2a98226fR10

https://github.com/WordPress/wordcamp.org/pull/33/commits/6560472e4740813d7bbf66613a625d417220432f#diff-bb376325754a536e359c629d2a98226fR17

Reply:
I like this. We could do this for `LABEL` as well. And I agree it would be better in index than in edit.

I'm not sure I understand why putting it in `index` didn't work. Couldn't you just export it there and then do `import { ORGANIZERS_ICON } from './index.js';` in `edit`?

Reply:
Huh, I guess you can. For some reason that didn't work the first time I tried it, but I must have done it wrong. Thanks!

Reply:
Done in 59cdcb645cd6dd62397ae527f2eaab5e53fdf104